step07.md

```diff
@@ -66,11 +66,11 @@ Try running the server again and reloading the page.
 
 You should see a very broken webpage now. There's a reason for that!
 
-When the handler function was in `server.js`, the paths to our assets (html, css etc) saved in the `pathToFile` and `pathToIndex` variables were correct. Now that the handler function has moved inside the `src` folder, we need to update the paths to our assets to make sure they point to the right location.
+When the handler function was in `server.js`, the paths to our assets (html, css etc) which saved in the `pathToFile` and `pathToIndex` variables, were correct. Now the handler function has moved inside the `src` folder, we need to update the paths to our assets to make sure they point to the right location.
 
-We can use **relative paths**. `..` means "go up one level to the folder above".  `.` means "look inside the current folder". For example: '../myComputer/documents' means "go up one folder, then go to 'myComputer', then go to 'documents'".
+We can use **relative paths**. `..` means "go up one level to the folder above".  `.` means "look inside the current folder". For example: '../myComputer/home.txt' means "go up one folder, then go to 'myComputer' folder, then go to 'home.txt' file". Another example: './home.txt' means "look inside the current folder for 'home.txt' file"
```

Comment:
For example: '../myComputer/home.txt' means "go up one folder, then go to *the* 'myComputer' folder, then go to *the* 'home.txt' file". Another example: './home.txt' means "look inside the current folder for *the* 'home.txt' file"